assembl/static2/js/app/components/debate/common/announcement.jsx

```diff
@@ -72,12 +84,16 @@ export const getSentimentsCount = (posts: Posts) => {
     );
 };
 
-export const createDoughnutElements = (sentimentCounts: SentimentsCounts) =>
-  Object.keys(sentimentCounts).map(key => ({
-    color: sentimentCounts[key].color,
-    count: sentimentCounts[key].count,
-    Tooltip: createTooltip(sentimentCounts[key], sentimentCounts[key].count)
-  }));
+export const createDoughnutElements = (sentimentCounts: SentimentsCounts): Array<DoughnutElements> =>
+  Object.keys(sentimentCounts).map(
+    key =>
+      sentimentCounts[key] && {
+        color: sentimentCounts[key].color,
+        count: sentimentCounts[key].count,
+        // $FlowFixMe flow believes sentimentCounts[key] can be undefined but it's not annotated as possibly undefined
```

Comment:
Do you still need this FlowFixMe with the `sentimentCounts[key]` check above and after reverting the SentimentsCounts changes?

Reply:
Yes